src/applications/vaos/components/LoadingOverlay.jsx

```diff
@@ -1,13 +1,21 @@
 import React, { useEffect } from 'react';
 import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+import recordEvent from 'platform/monitoring/record-event';
+
+let openTime;
 
 export default function LoadingOverlay({ show, message }) {
   useEffect(
     () => {
       if (show) {
         document.body.classList.add('modal-open');
+        openTime = Date.now();
```

Comment:
What made you choose to time this based on the overlay, vs the calls in the action?

I think this will work, just slightly suspicious of holding some state in `openTime`, outside the component.

Reply:
I did this mainly because it was more straightforward than timing the multiple API calls, figuring out the min/maxes of each, and figuring out which ended last. I could potentially store `openTime` in a `useState` hook instead if you think that's better.

Reply:
I think this is fine, it shouldn't cause any problems unless we use the loading overlay multiple times on the same page, which doesn't make sense.